src/svgp.jl

```diff
@@ -0,0 +1,68 @@
+struct SVGP end
+
+"""
+    approx_posterior(::SVGP, fz::FiniteGP, q::MvNormal)
+
+Compute the approximate posterior [1] over the process `f = fz.f`, given inducing
+inputs `z = fz.x` and a variational distribution over inducing points `q(u)` where `u =
+f(z)`. The approximate posterior at test points ``x^*`` where ``f^* = f(x^*)``
+is then given by:
+
+```math
+q(f^*) = \\int p(f | u) q(u) du
+```
+which can be found in closed form.
+
+[1] - Hensman, James, Alexander Matthews, and Zoubin Ghahramani. "Scalable
+variational Gaussian process classification." Artificial Intelligence and
+Statistics. PMLR, 2015.
+"""
+
+function approx_posterior(::SVGP, fz::FiniteGP, q::MvNormal)
```

Comment:
Instead of `MvNormal` you could expect and `AbstractMvNormal` and instead of `q.μ` call `mean(q)`